Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "SkyCoords"
 uuid = "fc659fc5-75a3-5475-a2ea-3da92c065361"
 authors = "Kyle Barbary, Mosé Giordano, and contributors"
-version = "0.4.0"
+version = "0.5.0-DEV"
```

Comment:
I have plans for another PR after this one to allow string construction, which is mostly written already so it could make sense to push that one as 0.5.0